packages/syft/src/syft/server/server.py

```diff
@@ -914,6 +915,26 @@ def update_self(self, settings: ServerSettings) -> None:
             if attr is not Empty:
                 setattr(self, attr_name, attr)
 
+    # NOTE: Some workflows currently expect the settings to be available,
```

Comment:
weird, saw that the tests were failing before because of this. Is there a ticket to look into that?

Reply:
Yup, but it was closed due to the fact that we didn't know how to reproduce: https://github.com/OpenMined/Heartbeat/issues/1264